static/app/components/tabs/tab.tsx

```diff
@@ -48,6 +65,14 @@ function BaseTab(
       <FocusLayer orientation={orientation} />
       {rendered}
       <TabSelectionIndicator orientation={orientation} selected={isSelected} />
+      {to && (
+        <TabLink
```

Comment:
It doesn't seem semantically correct to render an empty anchor tag here. I think we want this to wrap the `rendered` content

Reply:
True - changed in https://github.com/getsentry/sentry/pull/40852/commits/65875260ae22255e376c66d2c24861e5c6904c51